core/src/main/java/com/google/zxing/aztec/AztecDetectorResult.java

```diff
@@ -31,16 +31,19 @@ public final class AztecDetectorResult extends DetectorResult {
   private final boolean compact;
   private final int nbDatablocks;
   private final int nbLayers;
+  private final int errorsCorrected;
 
   public AztecDetectorResult(BitMatrix bits,
                              ResultPoint[] points,
                              boolean compact,
                              int nbDatablocks,
-                             int nbLayers) {
+                             int nbLayers,
```

Comment:
Retain the existing constructor - easy, and just ensures there is no binary compatibility issue.

Reply:
Done. I also changed one of the test classes back to using the new (old) constructor, so that both constructors are exercised during the tests.